README.md

```diff
@@ -1 +1,51 @@
-# molindo-node-logging
\ No newline at end of file
+# molindo-node-logger
+
+A node.js logger that integrates well with the Molindo infrastructure.
+
+## Features
+ - Pretty prints messages in development mode.
+ - Prints JSON messages in production mode.
+ - Errors are printed to `stderr`, while all other levels are printed to `stdout`.
+ - In development only warnings and errors are printed, while in production all levels are printed.
```

Comment:
This is actually outdated. In development infos, warnings and errors are printed now. I downgraded HTTP requests from info to debug so they don't show up in development logs.